### PR TITLE
SAK-31490 : Removing Home tool will remove other tools renamed to Home

### PR DESCRIPTION
**Pull request by frasese**
I don't know if this fix will cause more bugs than it solves. Someone can confirm, please.

### CODE REVIEW

**Comment by buckett** [comment]
Don't old installations still have home pages without the property set on them, in which case you will get an unremovable home page.

**Comment by mpellicer** [comment]
Thanks for your comments @buckett, we'll take a look to improve this, IMO deleting pages by page name is an annoying behaviour.

**Comment by buckett** [comment]
We have 109 home pages like this like this, but they all have the lesson builder tool in it.

`SELECT p.title, t.registration FROM SAKAI_SITE_PAGE p INNER JOIN SAKAI_SITE_TOOL t ON p.page_id = t.page_id  WHERE p.title = 'Home' and p.page_id NOT IN (SELECT page_id from SAKAI_SITE_PAGE_PROPERTY WHERE name = 'is_home_page' and value = 'true');
`
So from that point of view this looks ok.

File: site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java [hunk 1]
```diff
@@ -15325,16 +15325,11 @@ private String prefixString(String prefix, String aliasId) {
...
-			return true;
-		}
-		else
-		{
-			// if above fails, check based on the page title
-			String pageTitle = page.getTitle();
-			return TOOL_ID_HOME.equalsIgnoreCase(pageTitle) || rb.getString("java.home").equalsIgnoreCase(pageTitle);
+		//removed "check by title" : that creates unexpected results with normal pages titled as "HOME"
+		try {
+			return page.getProperties().getBooleanProperty(SitePage.IS_HOME_PAGE);
```
Comment (buckett) [inline site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java]: I think you can just do:

    return page.isHomePage()

and drop the try/catch.

**Comment by frasese** [comment]
@buckett you are right, doing page.isHomePage() is enough.

**Pull request merged**